src/anm/utils.js

```diff
@@ -252,6 +252,31 @@ function postpone(fn) {
     setTimeout(fn, 0);
 }
 
+function makeApiUrl(path, loadSrc) {
+    var prodHost = 'animatron.com',
+        testHost = 'animatron-test.com',
```

Comment:
I don't like it very much too. I only copied the code, and didn't want to touch it very much, so I won't break anything.